trackintel/analysis/location_identification.py

```diff
@@ -46,7 +46,8 @@ def location_identifier(spts, method="FREQ", pre_filter=True, **pre_filter_kwarg
     --------
     >>> ti.analysis.location_identifier(spts, pre_filter=True, method="FREQ")
     """
-    assert spts.as_staypoints
+    # check the correctness
```

Comment:
Hm... I think this comment is maybe not specific enough. In the ones below you wrote "check the correctness of the staypoints", I like that more because the reader immediately knows what is going on.
I think "correctness" is OK but I am not a big fan. I would prefer using "valid" 
Maybe "check if staypoints are valid" or "assert validity of staypoints"
(but as I said, I am OK with "correctness as long as it is changed to "correctness of staypoints" as you did below.